src/DbConnection.cpp

```diff
@@ -110,26 +102,21 @@ void DbConnection::check_connection() {
 }
 
 List DbConnection::info() {
-  return
-    List::create(
+  return List::create(
       _["host"] = std::string(pConn_->host),
       _["username"] = std::string(pConn_->user),
       _["dbname"] = std::string(pConn_->db ? pConn_->db : ""),
       _["con.type"] = std::string(mysql_get_host_info(pConn_)),
       _["db.version"] = std::string(mysql_get_server_info(pConn_)),
       _["port"] = NA_INTEGER,
-      _["protocol.version"] = (int) mysql_get_proto_info(pConn_),
-      _["thread.id"] = (int) mysql_thread_id(pConn_)
-    );
+      _["protocol.version"] = (int)mysql_get_proto_info(pConn_),
+      _["thread.id"] = (int)mysql_thread_id(pConn_));
```

Comment:
Can we move the closing paren `);` to the next line?

Reply:
This feature will be added to clang-format. Seems now doesn't exists.